internal/alloydbutil/engine_test.go

```diff
@@ -3,9 +3,83 @@ package alloydbutil
 import (
 	"context"
 	"errors"
+	"os"
 	"testing"
 )
 
+func getEnvVariables(t *testing.T) (string, string, string, string, string, string, string) {
+	t.Helper()
+
+	username := os.Getenv("ALLOYDB_USERNAME")
+	if username == "" {
+		t.Skip("ALLOYDB_USERNAME environment variable not set")
+	}
+	password := os.Getenv("ALLOYDB_PASSWORD")
+	if password == "" {
+		t.Skip("ALLOYDB_PASSWORD environment variable not set")
+	}
+	database := os.Getenv("ALLOYDB_DATABASE")
+	if database == "" {
+		t.Skip("ALLOYDB_DATABASE environment variable not set")
+	}
+	projectID := os.Getenv("ALLOYDB_PROJECT_ID")
+	if projectID == "" {
+		t.Skip("ALLOYDB_PROJECT_ID environment variable not set")
+	}
+	region := os.Getenv("ALLOYDB_REGION")
+	if region == "" {
+		t.Skip("ALLOYDB_REGION environment variable not set")
+	}
+	instance := os.Getenv("ALLOYDB_INSTANCE")
+	if instance == "" {
+		t.Skip("ALLOYDB_INSTANCE environment variable not set")
+	}
+	cluster := os.Getenv("ALLOYDB_CLUSTER")
+	if cluster == "" {
+		t.Skip("ALLOYDB_CLUSTER environment variable not set")
+	}
+
+	return username, password, database, projectID, region, instance, cluster
+}
+
+func setEngine(t *testing.T) (PostgresEngine, error) {
+	username, password, database, projectID, region, instance, cluster := getEnvVariables(t)
+	ctx := context.Background()
+	pgEngine, err := NewPostgresEngine(ctx,
+		WithUser(username),
+		WithPassword(password),
+		WithDatabase(database),
+		WithAlloyDBInstance(projectID, region, cluster, instance),
+	)
+	if err != nil {
+		t.Fatal("Could not set Engine: ", err)
```

Comment:
This function doesn't ever actually return an error -- just t.Fatal. 

It should be either:
1. remove the function and just do it in the test instead 
2. refactored to not need testing.T and to return an error
3. refactored to call t.Helper and not return an error

I'm inclined to #1 given it's limited use -- I don't think abstracting to a function is necessary.

Reply:
done!